reco_utils/recommender/ncf/ncf_singlenode.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Thanks for your explanation of the notation of `_singlenode`. Because this implementation in Tensorflow run on CPU or single GPU, so I use this name `_singlenode` from SAR. I think it makes sense if other versions of ncf are planned.

Reply:
makes sense